src/verilog/expr2verilog.cpp

```diff
@@ -1020,14 +1020,20 @@ std::string expr2verilogt::convert_constant(
     auto i = numeric_cast_v<mp_integer>(src);
 
     if(i>=256)
-      dest="'h"+integer2string(i, 16);
+    {
+      dest = std::to_string(width);
+      dest = "'h";
```

Comment:
The second write will just replace the first value - should this be `+=` instead? Also, this suggests a test case is missing here?

Reply:
Indeed -- fixed, and test added.

Reply:
Err, sorry, one of the previously existing test now needs to have its pattern adjusted (rightly so).